src/main/java/mate/jdbc/dao/DriverDaoImpl.java

```diff
@@ -0,0 +1,112 @@
+package mate.jdbc.dao;
```

Comment:
create an "impl" package and move the implementation there
```suggestion
package mate.jdbc.dao.impl;
```